constraints/cpu.md

```diff
@@ -0,0 +1,166 @@
+# CPU constraints in Kata Containers
+
+## Default number of virtual CPUs
+
+Before starting a container, the [runtime][6] reads the `default_vcpus` option
+from the [configuration file][7] to determine the number of virtual CPUs
+(vCPUs) needed to start the virtual machine. By default, `default_vcpus` is
+equal to 1 for fast boot time and a small memory footprint per virtual machine.
+Be aware that increasing this value negatively impacts the virtual machine's
+boot time and memory footprint.
+In general, we recommend that you do not edit this variable, unless you know
+what are you doing. If your container needs more than one vCPU, use
+[docker `--cpus`][1], [docker update][4] or [kubernetes `cpu` limits][2] to
```

Comment:
Added series comma: [docker `--cpus`][1], [docker update][4], or [kubernetes `cpu` limits][2] to